client/src/App.css

```diff
@@ -3,6 +3,7 @@
 @import url("./styles/_colors.css");
 @import url("./styles/Submitform.css");
 @import url("./styles/Home.css");
+@import url("./styles/Header.css");
```

Comment:
Actually, with `cra` and create react app is much better to load each css file in each component `js` file.

Reply:
ok thx